src/cluster/src/install.rs

```diff
@@ -555,6 +609,36 @@ impl ClusterInstaller {
         Ok(())
     }
 
+    /// Checks if all of the prerequisites for installing Fluvio are met
+    ///
+    /// This will attempt to automatically fix any missing prerequisites,
+    /// depending on the installer configuration. See the following options
+    /// for more details:
+    ///
+    /// - [`with_system_chart`]
+    /// - [`with_update_context`]
+    ///
+    /// [`with_system_chart`]: ./struct.ClusterInstaller.html#method.with_system_chart
+    /// [`with_update_context`]: ./struct.ClusterInstaller.html#method.with_update_context
+    fn check_and_fix(&self) -> Result<(), ClusterError> {
+        // Preforms pre-install checks and tries to fix things if possible
+        match self.pre_install_check() {
```

Comment:
let's move this under `pre_install_check` so we consolidate it `check and fix` features